modules/booksFunction.js

```diff
@@ -0,0 +1,56 @@
+import BookStore from './books.js';
+
+const newBookStorage = new BookStore();
+
+const bookStorage = JSON.parse(localStorage.getItem('bookCollection'));
+if (bookStorage !== null) {
+  newBookStorage.store = bookStorage;
+}
+
+export function loadBooksCollection() {
+  function loopingBook(book) {
+    return `<div id="${book.id}" class="book-card">
+      <p class="book-details">
+          "${book.title}" 
+          by 
+          ${book.author}
+      </p>
+      <button class="remove-button" data-book="${book.id}">Remove</button>
+      </div>`;
+  }
+  const bookContainer = document.querySelector('.books-container');
+  bookContainer.innerHTML = newBookStorage.store.map((book) => loopingBook(book)).join('');
+  // remove button
+  const removeButtons = document.querySelectorAll('.remove-button');
+  removeButtons.forEach((button) => {
+    button.addEventListener('click', () => {
+      const id = Number(button.getAttribute('data-book'));
+      newBookStorage.remove(id);
+      loadBooksCollection();
+      localStorage.setItem('bookCollection', JSON.stringify(newBookStorage.store));
+    });
+  });
+}
+
+export function loadContacts() {
```

Comment:
Please refactor this function to use arrow syntax to adhere to the project requirements🙏